app/views/coffee/info.html.erb

```diff
@@ -2,5 +2,5 @@
   <h2>More coming soon... </h2>
   <h3>For now, here's a guide to describing coffee flavor.</h3>
   
-  <img src="http://www.scaa.org/chronicle/wp-content/uploads/2016/01/SCAA_FlavorWheel.01.18.15.jpg" class="img-responsive" />
+  <img src="SCAA_FlavorWheel.jpg" class="img-responsive" />
```

Comment:
It's either gonna be `../../../assets/images/filename.jpg` or `../../assets/images/filename.jpg` try that and let me know @bermannoah